src/Expression.ts

```diff
@@ -32,7 +32,11 @@
  */
 import * as M from 'fp-ts/lib/Monoid'
 import * as T from 'fp-ts/lib/Traced'
-import { identity, Endomorphism, flow } from 'fp-ts/lib/function'
+import * as R from 'fp-ts/lib/Record'
+import * as S from 'fp-ts/lib/string'
+import * as B from 'fp-ts/lib/boolean'
+import { identity, Endomorphism, flow, pipe } from 'fp-ts/lib/function'
```

Comment:
Once `fp-ts@3.0.0` is released and we proceed with upgrading `expressive-ts`, we should update our `import` statements to follow the new pattern in `fp-ts` (i.e. `import * as R from 'fp-ts/Record` instead of `import * as R from 'fp-ts/lib/Record'`).

Reply:
Done